content/faq/content.md

```diff
@@ -0,0 +1,34 @@
+---
+title: What kind of content can I upload, and what about illegal or infringing content?
+category: publisher
+---
+
+This guide provides answers about questions regarding what you may upload to the LBRY app and network, as well as how to report illegal or infringing content.
+
+###What content can I legally upload to LBRY?
+
+You may upload content you created or own that does not infringe on the rights or safety of others, or content you have permission to upload. This includes Public Domain content, Creative Commons or Open Source content shared according to the license requirements, and Copyrighted content where you have permission from the owner.
```

Comment:
Do we have the say "safety" here? Is that against the law?

(I'm all for discouraging people from uploading content that would make others unsafe, but the question specifically says "what content can I legally upload?")

Reply:
I think by "safety" I was really referring to illegal content - like CP, threats, etc. Will modify.